esx_service/cli/vmdkops_admin.py

```diff
@@ -1301,16 +1305,38 @@ def config_rm(args):
     # This asks for double confirmation, and removes the local link or DB (if any)
     # NEVER deletes the shared database - instead prints help
 
-    if not args.local:
+    if not args.local and not args.unlink:
         return err_out("""
         Shared DB removal is not supported. For removing  local configuration, use --local flag.
-        For removing shared DB,  run 'vmdkops_admin config rm --local' on ESX hosts using this DB,
+        For removing shared DB,  run 'vmdkops_admin config rm --unlink' on ESX hosts using this DB,
         and manually remove the {} file from shared storage.
```

Comment:
I think the message no longer matches the functionality. 
Here it should say:
"""
DB removal is irreversible operation. Please use `--local` flag for removing DB in SingeNode mode, 
and use --unlink to unlink from DB in MultiNode mode". 
Note that --unlink will not remove a shared DB, but simply configure the current ESXi host to stop using it. 
For removing shared DB, run 'vmdkops_admin config rm --unlink' on ESXi hosts using this DB, and then manually remove the actual DB file from shared storage."""